PocketKit/Sources/PocketKit/Home/SlateDetailViewModel.swift

```diff
@@ -111,7 +112,8 @@ extension SlateDetailViewModel {
         let item = recommendation.item
         var destination: ContentOpen.Destination = .internal
         if item.shouldOpenInWebView {
-            let url = pocketPremiumURL(item.bestURL, user: user)
+            guard let bestURL = URL(string: item.bestURL) else { return }
```

Comment:
should we capture the url string if this does fail and send to Sentry?

Reply:
This'll change a bit once we're percent encoding every URL for its usage when used under certain circumstances (e.g presenting a web view), so logging can be added once that logic's in place. Good callout, though!